torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -263,6 +263,16 @@ The explanation needs to be clear on why this is needed. Here are some good exam
         "`-ic` flag is deprecated, please use `-i` instead for the same effect.";
     }
 
+    if (
+      ignore_current &&
+      !(await this.hasWorkflowRunningPermissions(
+        this.ctx.payload?.comment?.user?.login
+      ))
+    ) {
+      rejection_reason =
```

Comment:
Setting a rejection reason here should make trigger the conditional below to exit out early before even sending a workflow dispatch